src/main/scala/Robot.scala

```diff
@@ -8,58 +8,68 @@ object Directions {
   object Right extends Directions
 }
 
-class Robot {
-  var coordinates: (Int, Int) = (0, 0)
```

Comment:
Given the removal of the state of the Robot, it can become an `object` instead of a `class`.

Reply:
Done and done 👍